### DISCUSSION TITLE
MSFS-TouchPortal-Plugin not working

### DISCUSSION THREAD

**Discussion by magicnorm** [mpaperno/MSFSTouchPortalPlugin#33] [Q&A] [answered]
The MSFS-TouchPortal-Plugin is not working for me, I tried with a few commands, but nothing.
I hope the log can help.
Thanks

- Norman

[MSFSTouchPortalPlugin20220713.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/9107047/MSFSTouchPortalPlugin20220713.log)

**Comment by mpaperno** [comment, marked as answer]
The log looks clean, no issues.  But I don't see where it's connecting to MSFS at all.  Have you set up a way to connect?

Here's my "Connect" button:

![image](https://user-images.githubusercontent.com/1366615/178867515-88150192-43b0-4815-84c4-7bc623f4e975.png)

And the green light icon (not sure it's in Arbi's icon pack)
![green_led](https://user-images.githubusercontent.com/1366615/178867638-8045c9a6-f21d-4458-a31f-9eb073a6901d.png)

And the whole button (remove the .txt extension and import into Touch Portal).
[MSFS_Connection_button.tpb.txt](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/9107235/MSFS_Connection_button.tpb.txt)

**Reply by magicnorm** [reply]
It is know working for me after adding the connect button. I thought that it was connecting automatically. 
Still nothing is working with the PMDG 737. Hope someone somewhere will have a solution.
Again, thanks for your help. 
-Norman

**Reply by mpaperno** [reply]
Glad that's sorted!

I added a Wiki page with a couple example Touch Portal pages for the plugin, in case that helps.

https://github.com/mpaperno/MSFSTouchPortalPlugin/wiki/Pages-Buttons-and-Graphics

Cheers,
-Max
